src/pyhf/modifiers/histosys.py

```diff
@@ -128,10 +130,13 @@ def __init__(
             ]
             for m in keys
         ]
-        self._histosys_mask = [
-            [[builder_data[m][s]['data']['mask']] for s in pdfconfig.samples]
-            for m in keys
-        ]
+        self._histosys_mask = default_backend.astensor(
```

Comment:
I understand why we have these for PyTorch, but could we add a comment above all of them that this is for PyTorch only? Though being explicit about the `dtype` isn't a bad thing in general, I forget how much of penalty we get by calling `astensor` on stuff. I know that the backends are efficient about it, but I don't remember the Python overhead. If it is possible to drop this if we drop PyTorch that might be a good idea?